test/yaml/akri-opcua.yaml

```diff
@@ -4,14 +4,17 @@ metadata:
   name: akri-opcua
 spec:
   protocol:
-    opcua:
-      opcuaDiscoveryMethod: 
-        standard:
-          discoveryUrls:
-          - opc.tcp://localhost:4840/
-      applicationNames:
-        action: Include
-        items: []
+    name: opcua
+    discoveryDetails:
+      protocolHandler: |+
+        opcua:
```

Comment:
super cool that you found a way to keep some structured here !!!

Reply:
or are we parsing XXX under protocolHandler: opcua: XXX as a string?

Reply:
We are parsing the entire string value of discoveryDetails.get("protocolHandler").